1.Suppose a child psychologist says that the average time that working mothers spend talking to their children is at-least
11 minutes per day. You want to test
You conduct a random sample of 1000 working mothers and find they spend an average of 11.5 minutes per day talking with 
their children. Assume prior research suggests the population standard deviation is 2.3 minutes.
Conduct test with a level of significance of alpha=0.05

solution:

## State the null hypothesis (H0) and the alternative hypothesis (Ha):

 Null Hypothesis (H0): The average time working mothers spend talking to their children is 11 minutes or more per day .
 Alternative Hypothesis (Ha): The average time working mothers spend talking to their children is less than 11 minutes per day .
 
 
## Select the Significance level (alpha): 

 Given (alpha) = 0.05, this is the probability of making a Type I error (rejecting the null hypothesis when it's true).
 
 
##  Collect and Summarize Data

 Given data:

 Sample mean,x = 11.5 minutes

 Population standard deviation = 2.3 minutes

 Sample size = 1000
 
## Calculate the test statistic (z):
 

In [2]:
 mu =11
 x =11.5
 std =2.3
 z= (x- mu)/ std
 z

0.2173913043478261

## Determine the critical value:

  For a one-tailed test at a 0.05 level of significance, the critical z-value is approximately 1.645.
 

## Make a decision:

 Compare the calculated test statistic (z = 0.2174) to the critical z-value (1.645).
 Since the calculated test statistic is less than the critical z-value, you fail to reject the null  hypothesis.

## Draw a conclusion:

 Based on the sample data and the one-tailed z-test at a significance level of alpha = 0.05, there is insufficient evidence to  conclude that the average time working mothers spend talking to their children is less than 11 minutes per day. The  psychologist's claim is not supported by the sample data.
 The conclusion is that the average time spent talking to children, based on the sample data, is not significantly less than 11 minutes per day.
 